be/src/storage/vectorized/chunk_aggregator.h

```diff
@@ -87,6 +88,10 @@ class ChunkAggregator {
     std::vector<uint32_t> _aggregate_loops;
 
     ChunkPtr _aggregate_chunk;
+    // the last row of non-key column is in aggregator (not in aggregate chunk) before finalize.
+    // in vertical compaction, there maybe only non-key column in aggregate chunk,
+    // so we cannot use _aggregate_chunk->num_rows() as aggregated rows.
+    uint32_t _aggregated_rows = 0;
```

Comment:
what is the difference between _aggregate_rows and _aggregated_rows?

Reply:
Update to more clear names